prototypes_exercise/prototypes.js

```diff
@@ -0,0 +1,110 @@
+function Person(firstName, lastName, favoriteColor, favoriteNumber) {
+  this.firstName = firstName;
+  this.lastName = lastName;
+  this.favoriteColor = favoriteColor;
+  this.favoriteNumber = favoriteNumber;
+  this.favoriteFoods = [];
+  this.family = [];
+}
+
+Person.prototype.fullName = function() {
+  return this.firstName + " " + this.lastName;
+}
+
+var p = new Person("Shana", "Malarkin", "Green", 38);
+console.log(p.fullName()); // Shana Malarkin
+
+Person.prototype.toString = function() {
+  return this.firstName + " " + this.lastName + ", Favorite Color: " + this.favoriteColor + 
+          ", Favorite Number: " + this.favoriteNumber;
+}
+
+//var p1 = new Person("Shana", "Malarkin", "Green", 38);
+console.log(p.toString()); // Shana Malarkin, Favorite Color: Green, Favorite Number: 38
+
+Person.prototype.addToFamily = function(object) {
+  if (object instanceof Person)
+    if (this.family.indexOf(object) === -1)
```

Comment:
It's a matter of preference but I prefer curly braces around these if statements.  That way you can't accidentally forget the curly if you add more lines later.